packit_service/worker/helpers/testing_farm.py

```diff
@@ -165,6 +178,59 @@ def _pr(self) -> Optional[PullRequest]:
             self.__pr = self.project.get_pr(int(self.metadata.pr_id))
         return self.__pr
 
+    @property
+    def comment_command_parts(self) -> Optional[List[str]]:
+        """
+        List of packit comment command parts if the testing farm was triggered by a comment.
+
+        Example:
+            '/packit test' -> ["test"]
+            '/packit test namespace/repo#pr' -> ["test", "namespace/repo#pr"]
+        """
+        if not self._comment_command_parts and (
+            comment := self.metadata.event_dict.get("comment")
+        ):
+            self._comment_command_parts = get_packit_commands_from_comment(
+                comment,
+                packit_comment_command_prefix=self.service_config.comment_command_prefix,
+            )
+        return self._comment_command_parts
```

Comment:
I know it's a bite late for this comment (and probably I'm not the most qualified person to suggest this), but this method looks a bit strange, specially if because there is a mutation happening to the `self._comment_command_parts` class-property, and we are returning the value we just assigned to a class-property that can be accessed later. 

Ideally, I think it would be more readable and concise if this method doesn't return any result in it, but instead, override the value if necessary of the class-property and that's it. 

My suggestion for this one would be removing the return statement at the end of the method and leaving the mutation where it is right now, especially because the class initializes the property as `None`. 

```suggestion
    def comment_command_parts(self) -> Optional[List[str]]:
        """
        List of packit comment command parts if the testing farm was triggered by a comment.
        
        Note: 
            If the `self._comment_command_parts is not None, we will let the value as it is for now. 
            This method will only override the contents of the class-property if the same is `None`, and 
            we detect a comment from the `self.metadata` 
            
        Example:
            '/packit test' -> ["test"]
            '/packit test namespace/repo#pr' -> ["test", "namespace/repo#pr"]
        """
        if not self._comment_command_parts and (
            comment := self.metadata.event_dict.get("comment")
        ):
            self._comment_command_parts = get_packit_commands_from_comment(
                comment,
                packit_comment_command_prefix=self.service_config.comment_command_prefix,
            )
```

If you choose that, then you necessarily don't need to transform this method into a property, but rather, leave it as a method. 

If you would still prefer to have it as a property because that would trigger a lot of changes in other parts of the code, then, I think something like this could also be good: 

```suggestion
    def comment_command_parts(self) -> Optional[List[str]]:
        """
        List of packit comment command parts if the testing farm was triggered by a comment.
        
        Note: 
            If the `self._comment_command_parts is not None, we will let the value as it is for now. 
            This method will only override the contents of the class-property if the same is `None`, and 
            we detect a comment from the `self.metadata` 
            
        Example:
            '/packit test' -> ["test"]
            '/packit test namespace/repo#pr' -> ["test", "namespace/repo#pr"]
        """
        if comment := self.metadata.event_dict.get("comment"):
            return get_packit_commands_from_comment(
                comment,
                packit_comment_command_prefix=self.service_config.comment_command_prefix,
            )
       else: 
           return None
```

I'm recommending this especially because I can see that `self._comment_command_parts: Optional[List[str]] = None` is not used anywhere else other than here, so, probably, it would be easier to not have this class-property and have it either the method or the method-property instead. 

But of course, again, I may not be the best person to be proposing this right now, especially because the PR is already merged. Feel free to just close this comment if you think it's not worth it.

P.S: In my opinion, I lean towards the second suggestion.